sre-recipes/recipes/recipe-1/kubernetes-manifests/frontend.yaml

```diff
@@ -0,0 +1,98 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: frontend
+spec:
+  selector:
+    matchLabels:
+      app: frontend
+  template:
+    metadata:
+      labels:
+        app: frontend
+    spec:
+      containers:
+        - name: server
+          image: gcr.io/stackdriver-sandbox-230822/sandbox/frontend:recipe-1
```

Comment:
How is this image different from the normal frontend? Is it ok for this image to be frozen in time? (We won't update the tag unless it's part of the CI system)

Reply:
This image is specifically for the SRE recipes so I think it's okay for it to be frozen in time. If a developer wants to update the recipe then they'll have to update the image.

Reply:
Is it only the frontend you're freezing though? What if it becomes incompatible with the rest of the services? Are there tests in place to catch this?

Reply:
This is a good point - there wouldn't be any tests to catch this. I'll address this in your other comment